src/TSMapEditor/UI/UIManager.cs

```diff
@@ -408,8 +406,12 @@ private void CheckForIssuesAfterManualSave(object sender, EventArgs e)
 
                 string issuesString = string.Join(newline + newline, issues);
 
-                EditorMessageBox.Show(WindowManager, "Issues Found",
-                    "The map has been saved, but one or more issues have been found in the map. Please consider resolving them." + newline + newline + issuesString,
+                EditorMessageBox.Show(WindowManager, 
+                    Translate(this, "IssuesFound.Title", "Issues Found"),
+                    Translate(this, "IssuesFound.Description", 
+                        "The map has been saved, but one or more issues have been found in the map. Please consider resolving them." +
+                        newline + newline + newline + 
```

Comment:
You added one more newline than there was originally.

```suggestion
                        newline + newline +
```